sdk/template/Azure.Template/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 1.0.3-beta.2 (Unreleased)
-
+## 1.0.3-beta.2 (2020-09-04)
```

Comment:
If this version was released then we should also bump it again.

Reply:
The release tag is currently at beta.6. Will bump to beta.7